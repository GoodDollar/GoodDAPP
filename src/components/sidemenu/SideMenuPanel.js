// @flow
import React from 'react'
import { Platform, ScrollView, TouchableOpacity, View } from 'react-native'

import useSideMenu from '../../lib/hooks/useSideMenu'
import { withStyles } from '../../lib/styles'
import { ActionButton, Icon } from '../common'
import useOnPress from '../../lib/hooks/useOnPress'

//assets
import DiscordLogo from '../../assets/logos/socials/discord.png'
import DiscourseLogo from '../../assets/logos/socials/discourse.png'
import XLogo from '../../assets/logos/socials/Twitter-X.png'
import TgLogo from '../../assets/logos/socials/telegram.png'
import FacebookLogo from '../../assets/logos/socials/facebook.png'
import MediumLogo from '../../assets/logos/socials/medium.png'
import GdWebLogo from '../../assets/logos/socials/GdLogo.png'
import InstaLogo from '../../assets/logos/socials/instagram.png'
import LinkedinLogo from '../../assets/logos/socials/linkedin.png'

import SideMenuItem from './SideMenuItem'

export const socialIcons = {
  gdw: GdWebLogo,
  tg: TgLogo,
  x: XLogo,
  dsc: DiscourseLogo,
  inst: InstaLogo,
  dis: DiscordLogo,
  med: MediumLogo,
  fb: FacebookLogo,
  link: LinkedinLogo,
}

type SideMenuPanelProps = {
  navigation: any,
}

const SideMenuPanel = ({ navigation, styles, theme }: SideMenuPanelProps) => {
  const { slideToggle, topItems } = useSideMenu({
    navigation,
    theme,
  })

  const onPressClose = useOnPress(slideToggle)

  const socialItems = Object.entries(socialIcons)
  const firstRowItems = socialItems.slice(0, 4)
  const secondRowItems = socialItems.slice(4)

  return (
    <ScrollView contentContainerStyle={styles.scrollableContainer}>
      <TouchableOpacity style={styles.closeIconRow} onPress={onPressClose} testID="close_burger_button">
        <Icon name="close" size={20} color={theme.colors.lighterGray} />
      </TouchableOpacity>
      <View style={styles.listContainer}>
        {topItems.map(item => !item.hidden && <SideMenuItem key={item.name} {...item} />)}
        <View style={styles.iconsContainer}>
          <View style={styles.alignBottom}>
            {firstRowItems.map(([key, logo]) => (
              <ActionButton action={key} key={key} image={logo} />
            ))}
          </View>
          <View style={styles.alignBottom}>
            {secondRowItems.map(([key, logo]) => (
              <ActionButton action={key} key={key} image={logo} />
            ))}
          </View>
        </View>
      </View>
    </ScrollView>
  )
}

const sideMenuPanelStyles = ({ theme }) => ({
  scrollableContainer: {
    flexGrow: 1,
  },
  closeIconRow: {
    flex: -1,
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'flex-end',
    paddingTop: theme.sizes.defaultDouble,
    paddingBottom: theme.sizes.defaultQuadruple,
    marginHorizontal: theme.sizes.defaultDouble,
    minHeight: 20,
    ...Platform.select({
      web: { cursor: 'pointer' },
    }),
  },
  listContainer: {
    flexGrow: 1,
    borderTopWidth: 1,
    borderTopColor: theme.colors.lightGray,
    ...Platform.select({
      web: {
        borderTopStyle: 'solid',
      },
    }),
    marginHorizontal: theme.sizes.defaultDouble,
  },
  alignBottom: {
    ...Platform.select({
      web: {
        width: '100%',
        gap: 10,
      },
      android: {
        width: 250,
      },
    }),
    flexDirection: 'row',
    justifyContent: 'space-between',
  },
  iconsContainer: {
    marginTop: 'auto',
    justifyContent: 'center',
    alignItems: 'center',
    ...Platform.select({
      web: {
        width: '100%',
      },
    }),
    marginBottom: 32,
  },
})

export default withStyles(sideMenuPanelStyles)(SideMenuPanel)
