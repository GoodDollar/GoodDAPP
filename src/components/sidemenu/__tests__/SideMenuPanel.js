import React from 'react'
import renderer from 'react-test-renderer'
import SideMenuPanel from '../SideMenuPanel'
import { withThemeProvider } from '../../../__tests__/__util__'

const SideMenuPanelTheme = withThemeProvider(SideMenuPanel)

describe('SideMenuPanel', () => {
  it('matches snapshot', async () => {
    let component

    // eslint-disable-next-line require-await
    await renderer.act(async () => (component = renderer.create(<SideMenuPanelTheme />)))
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
