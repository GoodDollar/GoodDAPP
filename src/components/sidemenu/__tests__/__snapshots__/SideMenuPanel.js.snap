// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideMenuPanel matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n r-flex-13awgt0 r-pointerEvents-12vffkv"
>
  <div
    className="css-view-1dbjc4n r-WebkitOverflowScrolling-150rngu r-flexDirection-eqz5dr r-flexGrow-16y2uox r-flexShrink-1wbh5a2 r-overflowX-11yh6sk r-overflowY-1rnoaur r-transform-1sncvnh"
    onScroll={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    onWheel={[Function]}
  >
    <div
      className="css-view-1dbjc4n r-flexGrow-16y2uox"
    >
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-cursor-1loqt21 r-flex-9c3h7m r-flexDirection-18u37iz r-justifyContent-17s6mgv r-marginHorizontal-1ydw1k6 r-minHeight-bt1l66 r-paddingBottom-11rk87y r-paddingTop-95jzfe r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
        data-testid="close_burger_button"
        onClick={[Function]}
        onContextMenu={[Function]}
        onKeyDown={[Function]}
        style={
          Object {
            "WebkitTransitionDuration": "0s",
            "transitionDuration": "0s",
          }
        }
        tabIndex="0"
      >
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "color": "rgba(163,163,163,1.00)",
              "fontFamily": "gooddollar",
              "fontSize": "20px",
              "fontStyle": "normal",
              "fontWeight": "normal",
            }
          }
        >
          
        </div>
      </div>
      <div
        className="css-view-1dbjc4n r-borderTopColor-18grtmt r-borderTopStyle-1efd50x r-borderTopWidth-5kkj8d r-flexGrow-16y2uox r-marginHorizontal-1ydw1k6"
      >
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-borderBottomColor-1fn3lry r-borderBottomWidth-qklmqi r-borderStyle-1phboty r-cursor-1loqt21 r-flexDirection-18u37iz r-justifyContent-1h0z5md r-padding-nsbfu8 r-paddingLeft-gy4na3 r-paddingRight-9aemit r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
          onClick={[Function]}
          onContextMenu={[Function]}
          onKeyDown={[Function]}
          style={
            Object {
              "WebkitTransitionDuration": "0s",
              "transitionDuration": "0s",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-justifyContent-1777fci r-marginLeft-1jkjb r-marginRight-acz6fz r-width-q1j0wu"
          >
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(0,175,255,1.00)",
                  "fontFamily": "gooddollar",
                  "fontSize": "14px",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
            >
              
            </div>
          </div>
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-1b43r93 r-fontWeight-majxgm r-lineHeight-qvk6io r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y r-width-13qz1uu"
            dir="auto"
          >
            WalletConnect
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-borderBottomColor-1fn3lry r-borderBottomWidth-qklmqi r-borderStyle-1phboty r-cursor-1loqt21 r-flexDirection-18u37iz r-justifyContent-1h0z5md r-padding-nsbfu8 r-paddingLeft-gy4na3 r-paddingRight-9aemit r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
          onClick={[Function]}
          onContextMenu={[Function]}
          onKeyDown={[Function]}
          style={
            Object {
              "WebkitTransitionDuration": "0s",
              "transitionDuration": "0s",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-justifyContent-1777fci r-marginLeft-1jkjb r-marginRight-acz6fz r-width-q1j0wu"
          >
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(0,175,255,1.00)",
                  "fontFamily": "gooddollar",
                  "fontSize": "23px",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
            >
              
            </div>
          </div>
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-1b43r93 r-fontWeight-majxgm r-lineHeight-qvk6io r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y r-width-13qz1uu"
            dir="auto"
          >
            Go to GoodSwap
          </div>
          <div
            className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-justifyContent-1777fci r-marginLeft-1jkjb r-width-q1j0wu"
          >
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(0,175,255,1.00)",
                  "fontFamily": "gooddollar",
                  "fontSize": "23px",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
            >
              
            </div>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-borderBottomColor-1fn3lry r-borderBottomWidth-qklmqi r-borderStyle-1phboty r-cursor-1loqt21 r-flexDirection-18u37iz r-justifyContent-1h0z5md r-padding-nsbfu8 r-paddingLeft-gy4na3 r-paddingRight-9aemit r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
          onClick={[Function]}
          onContextMenu={[Function]}
          onKeyDown={[Function]}
          style={
            Object {
              "WebkitTransitionDuration": "0s",
              "transitionDuration": "0s",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-justifyContent-1777fci r-marginLeft-1jkjb r-marginRight-acz6fz r-width-q1j0wu"
          >
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(0,175,255,1.00)",
                  "fontFamily": "gooddollar",
                  "fontSize": "23px",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
            >
              
            </div>
          </div>
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-1b43r93 r-fontWeight-majxgm r-lineHeight-qvk6io r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y r-width-13qz1uu"
            dir="auto"
          >
            My profile
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-borderBottomColor-1fn3lry r-borderBottomWidth-qklmqi r-borderStyle-1phboty r-cursor-1loqt21 r-flexDirection-18u37iz r-justifyContent-1h0z5md r-padding-nsbfu8 r-paddingLeft-gy4na3 r-paddingRight-9aemit r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
          onClick={[Function]}
          onContextMenu={[Function]}
          onKeyDown={[Function]}
          style={
            Object {
              "WebkitTransitionDuration": "0s",
              "transitionDuration": "0s",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-justifyContent-1777fci r-marginLeft-1jkjb r-marginRight-acz6fz r-width-q1j0wu"
          >
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(0,175,255,1.00)",
                  "fontFamily": "gooddollar",
                  "fontSize": "18px",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
            >
              
            </div>
          </div>
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-1b43r93 r-fontWeight-majxgm r-lineHeight-qvk6io r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y r-width-13qz1uu"
            dir="auto"
          >
            Export Wallet
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-borderBottomColor-1fn3lry r-borderBottomWidth-qklmqi r-borderStyle-1phboty r-cursor-1loqt21 r-flexDirection-18u37iz r-justifyContent-1h0z5md r-padding-nsbfu8 r-paddingLeft-gy4na3 r-paddingRight-9aemit r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
          onClick={[Function]}
          onContextMenu={[Function]}
          onKeyDown={[Function]}
          style={
            Object {
              "WebkitTransitionDuration": "0s",
              "transitionDuration": "0s",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-justifyContent-1777fci r-marginLeft-1jkjb r-marginRight-acz6fz r-width-q1j0wu"
          >
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(0,175,255,1.00)",
                  "fontFamily": "gooddollar",
                  "fontSize": "23px",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
            >
              
            </div>
          </div>
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-1b43r93 r-fontWeight-majxgm r-lineHeight-qvk6io r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y r-width-13qz1uu"
            dir="auto"
          >
            Backup Wallet
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-borderBottomColor-1fn3lry r-borderBottomWidth-qklmqi r-borderStyle-1phboty r-cursor-1loqt21 r-flexDirection-18u37iz r-justifyContent-1h0z5md r-padding-nsbfu8 r-paddingLeft-gy4na3 r-paddingRight-9aemit r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
          onClick={[Function]}
          onContextMenu={[Function]}
          onKeyDown={[Function]}
          style={
            Object {
              "WebkitTransitionDuration": "0s",
              "transitionDuration": "0s",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-justifyContent-1777fci r-marginLeft-1jkjb r-marginRight-acz6fz r-width-q1j0wu"
          >
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(0,175,255,1.00)",
                  "fontFamily": "gooddollar",
                  "fontSize": "23px",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
            >
              
            </div>
          </div>
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-1b43r93 r-fontWeight-majxgm r-lineHeight-qvk6io r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y r-width-13qz1uu"
            dir="auto"
          >
            Settings
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-borderBottomColor-1fn3lry r-borderBottomWidth-qklmqi r-borderStyle-1phboty r-cursor-1loqt21 r-flexDirection-18u37iz r-justifyContent-1h0z5md r-padding-nsbfu8 r-paddingLeft-gy4na3 r-paddingRight-9aemit r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
          onClick={[Function]}
          onContextMenu={[Function]}
          onKeyDown={[Function]}
          style={
            Object {
              "WebkitTransitionDuration": "0s",
              "transitionDuration": "0s",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-display-6koalj r-justifyContent-1777fci r-marginLeft-1jkjb r-marginRight-acz6fz r-width-q1j0wu"
          >
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(0,175,255,1.00)",
                  "fontFamily": "gooddollar",
                  "fontSize": "18px",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
            >
              
            </div>
          </div>
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-1b43r93 r-fontWeight-majxgm r-lineHeight-qvk6io r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y r-width-13qz1uu"
            dir="auto"
          >
            Statistics
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-borderBottomColor-1fn3lry r-borderBottomWidth-qklmqi r-borderStyle-1phboty r-cursor-1loqt21 r-flexDirection-18u37iz r-justifyContent-1h0z5md r-padding-nsbfu8 r-paddingLeft-gy4na3 r-paddingRight-9aemit r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
          onClick={[Function]}
          onContextMenu={[Function]}
          onKeyDown={[Function]}
          style={
            Object {
              "WebkitTransitionDuration": "0s",
              "transitionDuration": "0s",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-justifyContent-1777fci r-marginLeft-1jkjb r-marginRight-acz6fz r-width-q1j0wu"
          >
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(0,175,255,1.00)",
                  "fontFamily": "gooddollar",
                  "fontSize": "18px",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
            >
              
            </div>
          </div>
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-1b43r93 r-fontWeight-majxgm r-lineHeight-qvk6io r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y r-width-13qz1uu"
            dir="auto"
          >
            Help & Feedback
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-borderBottomColor-1fn3lry r-borderBottomWidth-qklmqi r-borderStyle-1phboty r-cursor-1loqt21 r-flexDirection-18u37iz r-justifyContent-1h0z5md r-padding-nsbfu8 r-paddingLeft-gy4na3 r-paddingRight-9aemit r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
          onClick={[Function]}
          onContextMenu={[Function]}
          onKeyDown={[Function]}
          style={
            Object {
              "WebkitTransitionDuration": "0s",
              "transitionDuration": "0s",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-justifyContent-1777fci r-marginLeft-1jkjb r-marginRight-acz6fz r-width-q1j0wu"
          >
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(0,175,255,1.00)",
                  "fontFamily": "gooddollar",
                  "fontSize": "23px",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
            >
              
            </div>
          </div>
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-1b43r93 r-fontWeight-majxgm r-lineHeight-qvk6io r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y r-width-13qz1uu"
            dir="auto"
          >
            Privacy Policy & Terms
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-borderBottomColor-1fn3lry r-borderBottomWidth-qklmqi r-borderStyle-1phboty r-cursor-1loqt21 r-flexDirection-18u37iz r-justifyContent-1h0z5md r-padding-nsbfu8 r-paddingLeft-gy4na3 r-paddingRight-9aemit r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
          onClick={[Function]}
          onContextMenu={[Function]}
          onKeyDown={[Function]}
          style={
            Object {
              "WebkitTransitionDuration": "0s",
              "transitionDuration": "0s",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-justifyContent-1777fci r-marginLeft-1jkjb r-marginRight-acz6fz r-width-q1j0wu"
          >
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(0,175,255,1.00)",
                  "fontFamily": "gooddollar",
                  "fontSize": "23px",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                }
              }
            >
              
            </div>
          </div>
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-1b43r93 r-fontWeight-majxgm r-lineHeight-qvk6io r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y r-width-13qz1uu"
            dir="auto"
          >
            Logout
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-marginTop-1t01tom"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-borderBottomColor-1fn3lry r-borderBottomWidth-qklmqi r-borderStyle-1phboty r-cursor-1loqt21 r-flexDirection-18u37iz r-justifyContent-1h0z5md r-padding-nsbfu8 r-paddingLeft-gy4na3 r-paddingRight-9aemit r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
            onClick={[Function]}
            onContextMenu={[Function]}
            onKeyDown={[Function]}
            style={
              Object {
                "WebkitTransitionDuration": "0s",
                "transitionDuration": "0s",
              }
            }
            tabIndex="0"
          >
            <div
              className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-justifyContent-1777fci r-marginLeft-1jkjb r-marginRight-acz6fz r-width-q1j0wu"
            >
              <div
                className="css-text-901oao"
                dir="auto"
                style={
                  Object {
                    "color": "rgba(250,108,119,1.00)",
                    "fontFamily": "gooddollar",
                    "fontSize": "23px",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  }
                }
              >
                
              </div>
            </div>
            <div
              className="css-text-901oao r-color-11zaip3 r-fontFamily-vnw8o6 r-fontSize-1b43r93 r-fontWeight-majxgm r-lineHeight-qvk6io r-textAlign-fdjqy7 r-textDecorationLine-13wfysu r-textTransform-3twk1y r-width-13qz1uu"
              dir="auto"
            >
              Delete Account
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
